tests/integration/multi_table/test_hma.py

```diff
@@ -383,3 +384,98 @@ def test_hma_primary_key_and_foreign_key_only():
     # Assert
     assert all(sample['games']['user_id'].isin(sample['users']['user_id']))
     assert all(sample['games']['session_id'].isin(sample['sessions']['session_id']))
+
+
+def test_synthesize_multiple_tables_using_hma(tmpdir):
+    """End to end test for the ``SDV: Synthesize multiple tables (HMA).ipynb``."""
```

Comment:
Instead of referring to the demo can we summarize the parts being tested? ie:
- Anonymization
- Evaluating data
- Saving and loading